etc/c/odbc/sql.d

```diff
@@ -0,0 +1,1414 @@
+/**
+Declarations for interfacing with the ODBC library.
+
+Adapted with minimal changes from the work of David L. Davis
+(refer to the $(WEB
+forum.dlang.org/thread/cfk7ql$(DOLLAR)1p4n$(DOLLAR)1@digitaldaemon.com#post-cfk7ql:241p4n:241:40digitaldaemon.com,
+original announcement)).
+
+`etc.c.odbc.sql` is the the main include for ODBC v3.0+ Core functions,
+corresponding to the `sql.h` C header file. It `import`s `public`ly
+`etc.c.odbc.sqltypes` for conformity with the C header.
+
+See_Also: $(LUCKY ODBC API Reference on MSN Online)
+*/
+
+module etc.c.odbc.sql;
+
+public import etc.c.odbc.sqltypes;
+
+extern (Windows):

```

Comment:
I think  `extern(Windows)` would be the stdcall calling convention, which is different from Pascal [1].

[1] http://en.wikipedia.org/wiki/X86_calling_conventions#pascal